autofish.lua

```diff
@@ -169,14 +169,19 @@ local positionals = argparse.processArgsGetopt(args,
     handler=function() set_useRaw = not set_useRaw end}
 })
 
+load_state()
 if positionals[1] == "enable" then
     enabled = true
-    --print_status()
+
 elseif positionals[1] == "disable" then
     enabled = false
+    persist_state()
```

Comment:
Don't you need to cancel the repeat somewhere around here?

Reply:
don't timers auto-cancel when a map is unloaded when they aren't running per tick?

Reply:
What if the player disables the script with the `disable` command?